xamarin/Xamarin.Forms
# [UWP] Fix SwipeView executes Command twice

### Description of Change ###

Fix SwipeView executes Command twice on UWP. By assigning the Command to the native control (WinUI SwipeItem), the command was executed by the native control and by the OnInvoked method.

### Issues Resolved ### 

- fixes #11251 

### API Changes ###
 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

Not applicable

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 11251. Swipe to the right and tap the SwipeItem. Verify that the command has only been executed once (see Output).

### PR Checklist ###
